autotest/conftest.py

```diff
@@ -271,6 +271,23 @@ def pytest_report_header(config):
         if loaded_dlls:
             gdal_header_info += "\nLoaded shared objects:\n" + loaded_dlls
 
+    has_benchmark = False
+    for arg in sys.argv:
+        if "benchmark" in arg:
```

Comment:
This works if we call `pytest benchmark` from `autotest` but not if we call `pytest test_gtiff` from `autotest/benchmark`. See https://github.com/dbaston/gdal/commit/e1a77f48586a5416fe51b6a3bab3696dfe2c982d for an alternative that also writes a warning if we benchmark a debug build.

Reply:
thanks. Picked up